x-pack/platform/plugins/shared/osquery/common/api/model/schema/common_attributes.schema.yaml

```
@@ -51,66 +31,96 @@ components:
 
 
     Query:
+      description: 'The SQL query you want to run.'
+      example: 'select * from uptime;'
       type: string
 
     QueryOrUndefined:
       $ref: '#/components/schemas/Query'
       nullable: true
 
     Version:
+      description: 'Uses the Osquery versions greater than or equal to the specified version string.'
+      example: '1.0.0'
       type: string
 
     VersionOrUndefined:
       $ref: '#/components/schemas/Version'
       nullable: true
 
     Interval:
+      description: 'An interval, in seconds, on which to run the query.'
+      example: '60'
       type: string
 
     IntervalOrUndefined:
       $ref: '#/components/schemas/Interval'
       nullable: true
 
     Snapshot:
+      description: 'Indicates whether the query is a snapshot.'
+      example: true
       type: boolean
 
     SnapshotOrUndefined:
       $ref: '#/components/schemas/Snapshot'
       nullable: true
 
     Removed:
+      description: 'Indicates whether the query is removed.'
+      example: false
       type: boolean
 
     RemovedOrUndefined:
       $ref: '#/components/schemas/Removed'
       nullable: true
 
     PackName:
+      description: 'The pack name.'
       type: string
 
     SavedQueryId:
+      description: 'The ID of a saved query.'
+      example: '3c42c847-eb30-4452-80e0-728584042334'
       type: string
 
     SavedQueryIdOrUndefined:
       $ref: '#/components/schemas/SavedQueryId'
       nullable: true
 
+    SavedQueryDescription:
+      description: 'The saved query description.'
+      example: 'Saved query description'
+      type: string
+
+    SavedQueryDescriptionOrUndefined:
+      $ref: '#/components/schemas/SavedQueryDescription'
+      nullable: true
+
 
     PackId:
+      description: 'The ID of the pack you want to run.'
```

```suggestion
      description: 'The ID of the pack.'
```
since this description is used for multiple operations (e.g. delete, update)